pkg/kubernetes/scripts/tags.js

```diff
@@ -64,7 +64,7 @@
         return names;
     }
 
-    function buildSpec(names, spec) {
+    function buildSpec(names, spec, insecure) {
```

Comment:
With this there are now two tag-specific use cases that I see for this dialog:

1. adding or removing a tag
1. controlling if those tags are insecure

A registry is either insecure or secure so my assumption is that it applies to all images. I'm not sure we can support both use cases above without also passing in the state of the insecure flag.